schema/revert/database_functions/set_general_permissions.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ggircs-portal:set_general_permissions from pg
+
+begin;
+
+--TODO: Should we revert all permissions that are set by set_general_permissions here?
```

Comment:
if we're splitting this out, then we probably want to revert each grant in the revert file for the table/function

Reply:
any general ones that are left should probably be reverted here, yes